src/dcqc/tests/tests.py

```diff
@@ -142,3 +142,23 @@ def generate_process(self) -> Process:
             command_args=command_args,
         )
         return process
+
+class GrepDateTest(ExternalTestMixin, TestABC):
+    tier = 2
+
+    def generate_process(self) -> Process:
+        file = self._get_single_target_file()
+        path = file.local_path.as_posix()
+        command_args = [
+            "grep",
+            "-E",
+            "-w",
+            "-i",
+            "'date|time'",
+            path,
+        ]
+        process = Process(
+            container="quay.io/sagebionetworks/bftools:latest",
```

Comment:
I guess this can be any generic container?

Reply:
Agreed. You can use this one: 

```
quay.io/biocontainers/coreutils:8.30--h14c3975_1000
```

Reply:
Alternatively, we could perform this test in Python (_i.e._ an internal test). For this, we would drop the `ExternalTestMixin` parent class. If this is only intended to be performed on small files, then an internal test is slightly better because you can run it more easily outside of Nextflow.